bazel_tools/scala.bzl

```diff
@@ -652,3 +654,77 @@ def da_scala_benchmark_jmh(
     deps = resolve_scala_deps(deps, scala_deps, versioned_scala_deps)
     runtime_deps = resolve_scala_deps(runtime_deps, scala_runtime_deps)
     _wrap_rule_no_plugins(scala_benchmark_jmh, deps, runtime_deps, **kwargs)
+
+def to_camel_case(name):
```

Comment:
This file is mainly intended for very generic Scala rules. Maybe move this to something more specific in `//ledger/test-common`?

Reply:
You mean I can create a `bzl` file there and not in `bazel_tools`?

Reply:
exactly

Reply:
sure, will do